source/SIL.AppBuilder.Portal/src/lib/components/Dropdown.svelte

```diff
@@ -2,26 +2,32 @@
 @component
 A simple dropdown menu from DaisyUI.
 -->
+<!-- TODO: this component isn't used anywhere... -->
 <script lang="ts">
-  import { createEventDispatcher } from 'svelte';
-  export let cols = 6;
-  const dispatch = createEventDispatcher();
+  interface Props {
+    cols?: number;
+    label?: import('svelte').Snippet;
+    content?: import('svelte').Snippet;
```

Comment:
I think you should do:

`import { type Snippet } from 'svelte';`

and then use:

  interface Props {
    cols?: number;
    label?: Snippet;
    content?: Snippet;
    onNavEnd?: () => void;
  }